prov/verbs/src/fi_verbs.c

```diff
@@ -363,10 +373,20 @@ int fi_ibv_find_max_inline(struct ibv_pd *pd, struct ibv_context *context,
 {
 	struct ibv_qp_init_attr qp_attr;
 	struct ibv_qp *qp = NULL;
-	struct ibv_cq *cq = ibv_create_cq(context, 1, NULL, NULL, 0);
-	assert(cq);
+	struct ibv_cq *cq;
 	int max_inline = 2;
 	int rst = 0;
+	const char *dev_name = ibv_get_device_name(context->device);
+	uint8_t verbs_prov_num = count_of(verbs_prov_presets), i;
```

Comment:
No need for this variable since sizeof and hence count_of is calculated at compile time. count_of can be called directly in the for loop.